### ISSUE TITLE
@slide-change does not get fired in the new version 11

### ISSUE THREAD

**Issue by kiketordera** [nolimits4web/swiper#7181]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/wandering-meadow-vqwwng

### Bug description

The function provided in @slide-change does not get executed when the slider is changed to another slide

Steps to reproduce:
Install swipper version 11, create a function to be called on @slideChange and monitor his execution

Code example for .vue file:
```
<template>
  {{ executed }}
  <swiper-container
    :slides-per-view="3"
    effect="coverflow"
    :centered-slides="true"
    :autoplay="{ delay: 5000 }"
    :navigation="true"
    :pagination="{
      type: 'fraction',
    }"
    @slide-change="onSlideChange"
  >
    <swiper-slide>
      slide 1
    </swiper-slide>
    <swiper-slide>
      slide 2
    </swiper-slide>
    <swiper-slide>
      slide 3
    </swiper-slide>
  </swiper-container>
</template>

<script setup lang="ts">

const executed = ref(false)

function onSlideChange (e: any) {
  executed.value = true
  console.log('exacuted')
}

</script>
```

### Expected Behavior

The function provided in @slide-change should get executed when the slider is changed to another slide

### Actual Behavior

The function provided in @slide-change does not get executed when the slider is changed to another slide

### Swiper version

11.0.0

### Platform/Target and Browser Versions

Chrome latest

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by yuriystasiv** [comment]
I'm facing the same issue

**Comment by GNURub** [comment]
The problem is that the events now have a default prefix (swiper). I have the same problen in angular i have to do (swiperslidechange)="..." or you have to set the option {eventPrefix: ""}

**Comment by kiketordera** [comment]
Exactly, that made the trick @GNURub thanks! Maybe the examples in the documentation needs to be updated?

**Comment by Tanish2001** [comment]
In my case slideChangeTransitionEnd, reachBeginning and other events are not getting triggered, do i need add swiper at the start of these events too?

**Comment by radhouenchbinou** [comment]
Use swiperslidechange instead of slidechange

**Comment by WaqarKhan3** [comment]
I had this issue and wasn't able to figure out the problem since yesterday. Using swiperslidechange instead of slidechange did work. Thanks to @GNURub and @radhouenchbinou

**Comment by jgarciax** [comment]
> Utilice swiperslidechange en lugar de slidechange

GRAAAANDE, me salvaste
